docs/backend/global-utils.md

```diff
@@ -0,0 +1,253 @@
+---
+html_meta:
+  "description": "Global functions defined in CMFPlone and plone.app.layout."
+  "property=og:description": "Global functions defined in CMFPlone and plone.app.layout."
+  "property=og:title": "Global utils and helpers"
+  "keywords": "global variables,portal_state,context_state,plone_view,portal_url,helper methods"
+---
+
+
+# Global utils and helpers
+
+The following are global functions defined in `CMFPlone` and `plone.app.layout`.
+
+- `portal_state`
+- `context_state`
+- `plone_view`
+
+
+(backend-global-utils-plone-view-label)=
+
+## `plone_view`
```

Comment:
Were all of these docstrings copy-pasted from the source code? They have inconsistent formatting that Black would fix up. It would be good to have consistent formatting.

Reply:
it's like this in Plone core, CMFPlone, we have to fix it there first  ;)

Reply:
OK, that's what I thought. For this PR, this is fine. I created a new issue for them to resolve. https://github.com/plone/Products.CMFPlone/issues/3506